tests/app/notifications/rest/test_send_notification.py

```diff
@@ -254,7 +254,7 @@ def test_should_not_send_notification_if_restricted_and_not_a_service_user(notif
             assert response.status_code == 400
             assert [(
                 'Can’t send to this recipient when service is in trial mode '
-                '– see https://www.notifications.service.gov.uk/trial-mode'
+                f'– see {current_app.config["DOCUMENTATION_DOMAIN"]}/en/keys.html#live'
```

Comment:
Wondering if it would be better to use a hardcoded value in your tests, to prove that the value is indeed what you expect. I think that at the moment you're missing the `https://` part for example

Reply:
added https into DOCUMENTATION_DOMAIN